server/meeting_test.go

```diff
@@ -81,7 +81,7 @@ func TestPlugin_GenerateHashtag(t *testing.T) {
 					Schedule:      []time.Weekday{time.Monday},
 					HashtagFormat: "{{January 02 2006}}.vue",
 				}},
-			want:    "#" + assertNextWeekdayDate(time.Monday, false).Format("January 02 2006") + ".vue",
+			want:    "#" + strings.ReplaceAll(assertNextWeekdayDate(time.Monday, false).Format("January 02 2006"), " ", "_") + ".vue",
```

Comment:
We can use the underscores in the original formatting to avoid the string replace

```suggestion
			want:    "#" + assertNextWeekdayDate(time.Monday, false).Format("January_02_2006") + ".vue",
```

Reply:
Reason: `Format()` has special meaning for `_`.

Reply:
How are existing agenda settings affected by this change?